code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -80,10 +80,14 @@
 	var/deathmessage = ""
 	var/death_sound = null //The sound played on death
 
+	var/AIStatus = AI_ON //The Status of our AI, can be set to AI_ON (On, usual processing), AI_IDLE (Will not process, but will return to AI_ON if an enemy comes near), AI_OFF (Off, Not processing ever)
 

```

Comment:
should probably have `var/can_have_ai = TRUE //once we have become sentient, we can never go back` too.